internal/cmd/config/profile/set/set.go

```diff
@@ -49,6 +50,14 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				return fmt.Errorf("set profile: %w", err)
 			}
 
+			flow, err := auth.GetAuthFlow()
+			if err != nil {
+				p.Debug(print.WarningLevel, "both keyring and text file storage failed to find a valid authentication flow for the active profile")
+				p.Warn("Failed to find a valid authentication flow for the active profile. Please login using the 'stackit auth login' command.\n")
+			} else {
+				p.Debug(print.InfoLevel, "found valid authentication flow for active profile: %s", auth.GetPrettyAuthFlow(flow))
```

Comment:
do we have other INFO logs or is this the first one? We could consider no having INFO at all, just DEBUG, WARN or ERROR for simplicity

Reply:
That's true, we don't have it anywhere else. I'll simplify it then. But maybe we should at some point consider having this diversity to make the debug logs more readable